arthur_bench/scoring/hedging_language.py

```diff
@@ -20,6 +19,20 @@ class HedgingLanguage(Scorer):
     model output.
     """
 
+    def __init__(
+        self, model_type: str = DEFAULT_MODEL, hedging_language: str = DEFAULT_HEDGE
```

Comment:
It looks like `DEFAULT_HEDGE` is actually of type `Tuple[str]`. Since I believe this only makes since with a tuple of length 1, I think your instinct to type this as str makes sense. 

Would you mind updating this to accept a str, and transform to the correct dimension inside the intialization

Reply:
It's actually not a Tuple since there isn't a comma before the closing parenthesis. It seems rather a [PEP8 style](https://peps.python.org/pep-0008/?fbclid=IwAR3D5JcPI1uNSuIysRSjtXFKjhVQQr4WiA9qDJKtUA83u93jbfvqynlo2qg#:~:text=The%20preferred%20way%20of%20wrapping,a%20backslash%20for%20line%20continuation) for long strings. 

I guess we could change it for `"""` if you prefer to avoid mistaking it as a tuple

Reply:
Ah, I think it's ok to leave as is - thank you!